Dockerfile

```diff
@@ -21,7 +21,9 @@ COPY scripts/dev-entry.sh scripts/dev-entry.sh
 
 # Dev only
 RUN echo '!! If the next step fails, copy .env.example to .env in the backend folder !!'
-COPY .env .
+
+COPY .env* .
```

Comment:
I'm okay just copying over .env and .env.test individually. But this works too.

Reply:
What are the chances there will be other ones like `.env.prod` and `.env.stage`?

Reply:
None until there is 😎